conversation_reconstruction_dataflow/ingest_revisions/dataflow_main.py

```diff
@@ -96,17 +116,26 @@ def process(self, element):
 
     logging.info('USERLOG: Running ingestion process on %s' % chunk_name)
     ingestion_cmd = ['python2', '-m', 'ingest_utils.run_ingester', '-i', chunk_name]
+    kill = lambda process: process.kill()
     ingest_proc = subprocess.Popen(ingestion_cmd, stdout=subprocess.PIPE, bufsize = 4096)
-    # Yiqing's comment: stdout is the result of the ingestion
-    cnt = 0
-    maxsize = 0
-    for i, line in enumerate(ingest_proc.stdout):
-      ret = truncate_content(line)
-      for r in ret:
-          yield r
-      maxsize = max(maxsize, sys.getsizeof(line))
-      cnt += 1
-    logging.info('USERLOG: File %s complete! %s lines emitted, maxsize: %d' % (chunk_name, cnt, maxsize))
+    timer = Timer(my_timeout, kill, [ingest_proc])
```

Comment:
Lets not do this timer logic, it's hard to reason about, and potentially not helpful. Instead, as I suggested above, lets simply log with a certain frequency, then we can look at the logs to understand where things are at.

Reply:
Removed :)